### PR TITLE
Add options to overwrite range bounds in benchmark command.

### PR DESCRIPTION
**Pull request by marcio-diaz**
Small PR requested by @shawntabrizi  that adds the option to overwrite the range bounds for each component variable while using `benchmark` command.

### CODE REVIEW

File: frame/benchmarking/src/utils.rs [hunk 1]
```diff
@@ -77,8 +79,16 @@ pub trait Benchmarking<T> {
 	/// Parameters
 	/// - `extrinsic`: The name of extrinsic function you want to benchmark encoded as bytes.
 	/// - `steps`: The number of sample points you want to take across the range of parameters.
+	/// - `mins`: The minimum number for each range of parameters.
+	/// - `maxs`: The maximum number for each range of parameters.
 	/// - `repeat`: The number of times you want to repeat a benchmark.
-	fn run_benchmark(extrinsic: Vec<u8>, steps: Vec<u32>, repeat: u32) -> Result<Vec<T>, &'static str>;
+	fn run_benchmark(
+		extrinsic: Vec<u8>,
+		mins: Vec<u32>,
```
Comment (drahnr) [inline frame/benchmarking/src/utils.rs]: possibly rename `mins` to `min_number_of_steps`

File: frame/benchmarking/src/lib.rs [hunk 2]
```diff
@@ -165,18 +171,29 @@ macro_rules! impl_benchmark {
 					let steps = steps.get(idx).unwrap_or(&prev_steps);
 					prev_steps = steps;
 
+					let min = mins.get(idx).unwrap_or(&low);
```
Comment (drahnr) [inline frame/benchmarking/src/lib.rs]: `let min = mins.get(idx).cloned().unwrap_or(low);` looks more intuitive to me, then you can remove all `*` later on

**Comment by marcio-diaz** [comment]
> Nit picks :) looks good to me other than that

Thanks! Fixed.

**Comment by gnunicorn** [comment]
@marcio-diaz please update to master.

**Pull request merged**